projects/addon-doc/src/components/demo/demo.component.ts

```diff
@@ -31,6 +32,13 @@ import {TUI_DOC_URL_STATE_HANDLER} from '../../tokens/url-state-handler';
 
 const MIN_WIDTH = 160;
 
+interface TuiDefaultParams {
+    tuiMode?: 'onDark' | 'onLight' | null;
+    updateOn?: 'blur' | 'change' | 'submit';
+    sandboxExpanded?: boolean | string;
+    sandboxWidth?: number;
```

Comment:
Isn't it `string` in params? And `sandboxExpanded` is also `string`, not `boolean`?

Reply:
fixed